Source/MessagingTextInputbar.m

```diff
@@ -0,0 +1,115 @@
+//
+//  MessagingTextInputbar.m
+//  TrunkClub
+//
+//  Created by Matt Bush on 7/19/16.
+//  Copyright © 2016 Trunk Club. All rights reserved.
+//
+
+#import "MessagingTextInputbar.h"
+#import "SLKTextView+SLKAdditions.h"
+#import "UIView+SLKAdditions.h"
+
+CGFloat const kCustomToolbarHeight = 36.0;
+
+@interface SLKTextInputbar (Private)
+
+- (void) slk_setupViewConstraints;
+- (void) slk_commonInit;
+@property (nonatomic, retain) UIView *customToolbarView;
+@property (nonatomic, strong) NSLayoutConstraint *leftButtonWC;
+@property (nonatomic, strong) NSLayoutConstraint *leftButtonHC;
+@property (nonatomic, strong) NSLayoutConstraint *leftMarginWC;
+@property (nonatomic, strong) NSLayoutConstraint *bottomMarginWC;
+@property (nonatomic, strong) NSLayoutConstraint *rightButtonWC;
+@property (nonatomic, strong) NSLayoutConstraint *rightMarginWC;
+@property (nonatomic, strong) NSLayoutConstraint *rightButtonTopMarginC;
+@property (nonatomic, strong) NSLayoutConstraint *rightButtonBottomMarginC;
+@property (nonatomic, strong) NSLayoutConstraint *editorContentViewHC;
+
+@end
+
+@implementation MessagingTextInputbar
+
+@synthesize customToolbarView = _customToolbarView;
+
+- (CGFloat)appropriateHeight {
+    CGFloat height = super.appropriateHeight;
```

Comment:
curious about the syntax, is there a certain time and place you put the { on the same line whereas the other ones have it on the next line